zips/draft-ecc-community-and-coinholder.md

```diff
@@ -86,49 +84,7 @@ This funding stream will start on expiry of the existing ``FS_FPF_ZCG`` funding
-
-### Option 2: Revert to hard-coded output address
-
-The 12% of the block subsidy that currently is distributed to the lockbox via the existing ``FS_DEFERRED`` funding stream, is instead regularly distributed to a 3-of-5 P2SH multisig with keys held by the same Key-Holder Organizations as above. The Coinholder-Controlled Fund is considered to include both this stream of funds, and funds from the one-time lockbox disbursement described above.
-
-Option 2 can be realized by either of the following mechanisms:
-
-#### Mechanism 2a: Classic funding stream
-
-A new funding stream is defined that replaces the existing ``FS_DEFERRED`` lockbox funding stream [^zip-1015-funding-streams]. That is, the end height of ``FS_DEFERRED`` will be changed to the activation height of this ZIP. The new ``FS_COINHOLDER`` stream will start at the same height and last until the 3rd halving.
-
-#### Mechanism 2b: Periodic lockbox disbursement
-
-Constant parameter $N = 35000$ blocks $= \mathsf{PostBlossomHalvingInterval}/48$ (i.e. approximately one month of blocks).
-
-The ``FS_DEFERRED`` lockbox funding stream is extended by a further 1,260,000 blocks (approximately 3 years), ending at the 3rd halving. A consensus rule is added to disburse from the Deferred Dev Fund Lockbox to a 3-of-5 P2SH multisig with keys held by the same Key-Holder Organizations as above, starting at block height $\mathsf{activation\_height} + N$ and continuing at periodic intervals of $N$ blocks ending with the 3rd halving height. Each disbursement empties the lockbox. This is equivalent to specifying $\frac{\mathstrut\mathsf{third\_halving\_height} \,-\, \mathsf{activation\_height}}{N}$ [One-time lockbox disbursement]s, that all output to the same address.
-
-#### Rationale for periodic disbursement
-
-Classic funding streams [^zip-0207] produce many small output values, due to only being able to direct funds from a single block's subsidy at a time. This creates operational burdens to utilizing the funds — in particular due to block and transaction sizes limiting how many outputs can be combined at once, which increases the number of required transactions and correspondingly the overall fee.
-
-The periodic lockbox disbursement mechanism can produce the same effective funding stream, but with aggregation performed for free: the output to the funding stream recipient is aggregated into larger outputs every $N$ blocks. In the specific case of Mechanism 2b, the recipient multisig address would receive around 40 outputs, instead of around 1,300,000.
+The pool of multisig-controlled funds described as the Zcash Development Fund by [^draft-ecc-lockbox-distribution] forms a new Coinholder-Controlled Fund.
```

Comment:
Calling it the Zcash Development Fund is confusing because in ZIPs 1014 and 1015, that included the ZCG Slice.

Reply:
I don't know how to generalize this name, because in the community-and-coinholder proposal this fund receives 12% of the block reward, and in the ZBloc proposal it receives 20%.

Reply:
Figured it out.